Añade tanques de plasma al protolathe.

Evankhell561 opened the pull request:
La tecnologia de toxinas fue agregada al juego recientemente, ésta, a pesar de lo que me esperaba no trajo muchas cosas interesantes al juego o alguna recompensa por subir esta tecnologia. Este pr intenta arreglar esto dandoles cosas algunas facilidades a los apasionados de las bombas de toxinas, aunque no sea mucho realmente. He agregado tanques de plasma al protolathe y los crates de valvulas de transferencia ahora pueden ser abiertas por cientificos para que la fiesta de bombas no pare (la recompensa de su trabajo sera mas trabajo)


Actualizacion:
Ya no modifica nada del BRPD.
:cl:
add: Agrego tanques de plasma al protolathe.
/:cl:

Commit:
empty plasmatank guardados en hispania

esto guarda los plasma tank vacio en hispania para que no se pierdan los cambios cunado se actualice desde paradise.

Commit:
cosas nuevas para toxin tech

Agrego valvulas de transferencia y tanques de plasma al protolathe
balanceo el costo del bluespace RPD.

Commit:
Valvulas con acceso de cientifico

valvulas con acceso de cientifico, no mas en el protolathe.

Tilk1 commented:
El tanque de plasma vacio no tiene mucho sentido. Eso puede ser crafteado por cualquier persona con metal.

Ademas la protolathe no deberia fabricar objetos tan grandes y que tienen colision

Commit:
Agregados a hispania

agrego los cambios que hice en paradise de cosas nuevas para toxinas a la carpeta hispania.

Tilk1 commented:
El resto estoy todo de acuerdo

Evankhell561 commented:
> El tanque de plasma vacio no tiene mucho sentido. Eso puede ser crafteado por cualquier persona con metal.
> 
> Ademas la protolathe no deberia fabricar objetos tan grandes y que tienen colision

nica, de que hablas? objetos tan grandes y que tienen colision? los tanques de plasma no tienen colision (puedes pasar sobre ellos), no son grandes en el sentido de que puedes meter varios en la mochila y en el protolathe ya se pueden crear tanques de oxigeno que son basicamente lo mismo pero de una skin difente. Por no mencionar que acabo de buscar y no se pueden craftear tanques de plasma.

iRyzor commented:
Te has hecho un lío con los archivos

DrawitRalph commented:
Necesitaré testear esto a fondo antes de mergear porque el cambio de requisitos de materiales me preocupa. RnD se ha prestado para mucho abuso y pienso que recompensar eso no es el mejor plan de acción.

Evankhell561 commented:
> Necesitaré testear esto a fondo antes de mergear porque el cambio de requisitos de materiales me preocupa. RnD se ha prestado para mucho abuso y pienso que recompensar eso no es el mejor plan de acción.

vale, yo no planeaba reducir costes del bluespace RPD. de hecho me parece que ahora es mas dificil de obtener. lo que queria cambiar es la forma en la que se hace. normalmente no le metes a la consola muchisimo de un solo material sino que repartes capacidad del protholathe entre todos los materiales que tienes disponibles. si se ha reducido la cantidad total de minerales es porque obtener plata, oro, plasma, y plasteel es mas dificil que conseguir meatal y vidrio.

Evankhell561 committed:
Delete supplypacks.dm

Evankhell561 committed:
Delete tank_types.dm

Evankhell561 committed:
Delete bluespace_designs.dm

Evankhell561 committed:
Delete misc_designs.dm

Tilk1 commented:
Crei que te referias a los canisters my bad

Tilk1 commented:
Crei que te referias a los canisters my bad

Commit:
sube los requisitos del RPD

sube los costos del bluespace RPD

Evankhell561 commented:
subi aun mas los precios del bluespace rpd debido a la preocupacion de rafa. luego de meditarlo un poco creo que la reduccion de costos si era un poco bastante, ahora no se compensa nada. los valores son los mismos pero repartidos diferente.  espero que ahora satisfaga a todos. De ultimas, si da problemas lo del BRPD, estoy dispuesto a eliminar eso del pr.

Commit:
Merge branch 'master' into Toxinas

Commit:
sube los requisitos de minerales

Commit:
Merge branch 'master' into Toxinas

Commit:
Merge branch 'master' into Toxinas

Evankhell561 committed:
Merge branch 'master' into Toxinas

Commit:
Merge branch 'master' into Toxinas

Commit:
mueve los archivos para que quede bonito

Commit:
Merge branch 'Toxinas' of https://github.com/Evankhell561/Paradise into Toxinas

Commit:
Update paradise.dme

Dapple27 commented:
creo que deberias especificar en la descripcion a cuanto rebajaste el costo del RPD

Dapple27 commented:
por cierto , veo que añadiste mas materiales para el RPD , segun lo que veo 
![image](https://user-images.githubusercontent.com/43283605/56577423-fa487e00-6598-11e9-9c0d-b9beabc9ac43.png) 
eso es lo que deberias resaltar en la descripcion :+1:

Evankhell561 commented:
> por cierto , veo que añadiste mas materiales para el RPD , segun lo que veo

dana, no lo aumente, solo movi los costos de un lado a otro. la cantidad total es la misma

Dapple27 commented:
bueno , pero por lo que leo añadiste mas materiales para contruirlo :thinking:

Commit:
Merge branch 'master' into Toxinas

Commit:
Update paradise.dme

Commit:
Update paradise.dme

Evankhell561 committed:
Merge branch 'master' into Toxinas

Commit:
Update paradise.dme

Commit:
Merge branch 'master' into Toxinas

Evankhell561 commented:
> bueno , pero por lo que leo añadiste mas materiales para contruirlo

le añadi materiales para aconstuirlo porque le quite metal y vidrio. porque hice esto? la cantidad de metal y vidrio que pide son cantidades que usualmente no le metes al protholate. lo normal es meterle algo de cada material. no muchisimo de uno solo. pretende ser un cambio por comodidad y no por hacerlo mas dificil o facil de contruir en si.

Commit:
revierte los cambios al RPD

iRyzor commented:
Este PR fue testeado?

Commit:
mueve las cosas de mover.

Commit:
descripción fix.

Commit:
Merge branch 'master' into Toxinas

Evankhell561 commented:
> Este PR fue testeado?

ha sido testeado cada vez y siempre ha funcionado. si tiene tantos commits es por mi antigua mania de llevar los cambios que se le hacían rama principal a mi branch. Lo he vuelto a probar y sigue funcionando.

Commit:
porque el jefe manda.
